content/google-analytics/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Google Analytics
+content_type: project
+submission_type: link
+---
+
+Google Analytics for Beginners shows new users how to create an account, implement tracking code, and set up data filters. You'll learn how to navigate the Google Analytics interface and reports, and set up dashboards and shortcuts. The course will also demonstrate how to analyze basic Audience, Acquisition, and Behavior reports, and set up goals and campaign tracking.
+
+### Task
+Complete the Google Analytics for Beginners. This will roughly take 4-6 hours if completed sequentially. You have the choice of watching the videos or reading the transcripts. Complete all 4 short units and submit a screenshot of your certificate.

```

Comment:
You might want to give people a bit more info on how to submit their work...

Tilde doesn't allow folks to upload screenshots.  I'd suggest "take a screenshot of your certificate and upload it to google drive. Please share a link to your screenshot"

Trust me, people will be annoying if you aren't clear on this stuff!